x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -477,13 +507,18 @@ export class AlertsClient {
         throw Boom.notFound(errorMessage);
       }
 
+      const fieldToUpdate =
```

Comment:
Can you put this logic in a function like `getAlertStatusField` to be used here and in `bulkUpdateRequest`?